node/common/signature.ts

```diff
@@ -5,3 +5,8 @@ export interface SignatureSigner {
 export interface SignatureValidator {
   validate(data: Uint8Array, signature: Uint8Array): Promise<boolean>;
 }
+
+export interface SignatureSignerFactory {
```

Comment:
Node/Web SDK will implements the SignatureSignerFacotry (one is based on the eclipse package and the other one is based on WebCrypto)